frontend/src/components/transactions/__tests__/common.tsx

```diff
@@ -189,3 +337,194 @@ export const TRANSACTIONS: T.Transaction[] = [
     },
   },
 ];
+
+export const RECEIPT_OUTCOMES_BY_ID = [
+  // for those no action has deposit
```

Comment:
Instead of having a list of groups of receipt outcomes leading to quite a mysterious usage of `RECEIPT_OUTCOMES_BY_ID[0]` (`0` is not an id, and I was quite confused), I suggest you have separate constants `RECEIPT_OUTCOMES_BY_ID_WITH_NO_DEPOSIT_ACTIONS`, `RECEIPT_OUTCOMES_BY_ID_WITH_SMALL_DEPOSIT_ACTIONS`, ...